File: done/c/1118.c
#include <stdio.h>

int main(void)
{
    int novo_calculo = 1;

    while (novo_calculo == 1)
    {
        int contador = 2;
        float x, media = 0;

        while (contador)
        {
            scanf("%f", &x);

            if (x >= 0 && x <= 10)
            {
                media += x, contador--;
            }
            else
            {
                printf("nota invalida\n");
            }
        }

        printf("media = %.2f\n", media / 2.0);

        do
        {
            printf("novo calculo (1-sim 2-nao)\n");
            scanf("%d", &novo_calculo);
        }
        while (novo_calculo != 1 && novo_calculo != 2);
    }

    return 0;
}

File: problems/python/1118.py
# 1118 - Several Scores with Validation
nums_list = []
new_calculate = 1.0
while new_calculate != 2.0:
    counter = 1
    while counter < 3:
        n = float(input())
        if (n < 0) or (n > 10):
            print("nota invalida")
        else:
            nums_list.append(n)
            counter += 1
    average = (nums_list[0] + nums_list[1]) / 2
    print("media = {:.2f}".format(average))
    print("novo calculo (1-sim 2-nao)")
    new_calculate = float(input())
    while (new_calculate < 1.0) or (new_calculate > 2.0):
        print("novo calculo (1-sim 2-nao)")
        new_calculate = float(input())
    average = 0
    nums_list = []
